robot_state: '/state'
robot_topopath: '/topopath'
interest_points: '/interest_points'
afference: '/afference'
destinations_log: '/destinations_log'
pose_topic: '/amcl_pose'
goal_topic: '/move_base_simple/goal'
make_plan: '/move_base/NavfnROS/make_plan'
namespaces_topic: '/colors/'

robot_max_speed: 0.26
robot_state_rate: 2

ipoint_radius: 0.4

logging: true  # prints useful colored logs to stdout
simulation_confirm_gui: false
simulation_time_measure: 'minutes'  # can be 'minutes' or 'seconds'
#simulation_time_measure: 'seconds'
simulation_duration: 1  # time duration of the simulation, based on the time measure above